packages/block-library/src/shortcode/editor.scss

```diff
@@ -1,13 +1,33 @@
 [data-type="core/shortcode"] {
-	.block-editor-plain-text {
-		max-height: 250px;
-	}
-
 	&.components-placeholder {
 		min-height: 0;
 	}
 }
 
+// The editing view for the Shortcode block is equivalent to block UI.
+// Therefore we increase specificity to avoid theme styles bleeding in.
 .blocks-shortcode__textarea {
-	@include input-control;
+	font-family: $editor-html-font !important;
+	color: $gray-900 !important;
+	background: $white !important;
+	padding: $grid-unit-15 !important;
+	border: $border-width solid $gray-900 !important;
+	box-shadow: none !important;
+	border-radius: $radius-block-ui !important;
+	box-sizing: border-box;
```

Comment:
I included the `box-sizing` to contain a text area inside the placeholder for some themes, like 2023.

## Screeenshot
![CleanShot 2023-04-11 at 20 04 44](https://user-images.githubusercontent.com/240569/231223195-4556fd4d-30c9-42bb-ba62-4fa33c13fdc4.png)